src/core/crypto.cpp

```diff
@@ -234,6 +240,14 @@ SHA256Hash mineHash(SHA256Hash target, unsigned char challengeSize, bool usePuff
     for (size_t i = 32; i < 64; ++i) concat[i] = t_rand() % 256;
 
     while(true) {
+
+        if (hashes++ > 1024) {
+            double hps = (double)hashes / ((double)(get_time_millis() - st) / 1000);
+            Logger::logStatus("[thread" + std::to_string(thrid) + "] mining at " + std::to_string(hps) + "h/sec");
+            hashes = 0;
+            st = get_time_millis();
```

Comment:
Please use camel case convention "getTimeMilliseconds()"

Reply:
Resolved in  ebfa87e.